### PR TITLE
Update OWNERS, add cpmeadors

### PR DESCRIPTION
**Pull request by jensfr**
Add @cpmeadors as reviewer and approver

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -3,8 +3,10 @@ reviewers:
   - pmores
   - bpradipt
   - gkurz
+  - cmeadors
```
Comment (gkurz) [inline OWNERS:6]: s/cmeadors/cpmeadors

**Review by gkurz — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 1 above)
Comment (jensfr) [reply OWNERS:6]: oh, good catch, it's different on github and slack

**Review by gkurz — APPROVED** [review]
LGTM. Thanks @jensfr !

**Comment by gkurz** [comment]
/override  ci/prow/4.9-ci-index-sandboxed-containers-operator-bundle

**Comment by jensfr** [comment]
/override ci/prow/4.9-ci-index-sandboxed-containers-operator-bundle

**Comment by gkurz** [comment]
> /retest

@jensfr AFAIK these e2e checks always fail but they aren't required => you can merge right away (I sill don't have super powers)

**Comment by pmores** [comment]
Why is this two commits though?

**Comment by gkurz** [comment]
> Why is this two commits though?

Second commit is fixing the first one. I'd personally make this a single patch and force-push.

**Pull request merged**